src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespacebefore/InputNoWhitespaceBeforeMethodRef.java

```diff
@@ -0,0 +1,23 @@
+package com . puppycrawl
+    .tools.
+    checkstyle.checks.whitespace.nowhitespacebefore;
```

Comment:
please simplify package - make it single line.
in this input you should focus on NoWhitespaceBeforeMethodRef